spec/cnb/application_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+RSpec.describe CNB::Application do
+  let(:app)      { CNB::Application.new }
+
+  describe '#exchange_rate' do
+    before do
+      Timecop.freeze('2015-10-19')
+    end
+
+    after do
+      Timecop.return
+    end
+
+    context 'date missing' do
+      it 'calls exchange_rate_for with default date value(Date.today)' do
+        expect(app.today).to receive(:exchange_rate_for).with('AUD', Date.today)
+        app.exchange_rate('AUD')
+      end
+    end
+
+    context 'currency is not supported' do
+      it 'raises CurrencyNotSupported exception' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.